README.md

```diff
@@ -50,3 +51,8 @@ Test your new function; I've provided some basic testing code in `code.test.js`.
 What is the worst-case asymptotic time complexity of your implementation? What
 is the worst-case asymptotic memory complexity? Add your answer, including your
 reasoning, to this markdown file.
+
+## Answer
+For the beginning of my algorithm it creates a list (route) and that takes $n$ time it then randomizes/shuffles the list and that also takes $n$ time. So we have $n^2$ for now. In the part of the code that runs the twoOptSwap it runs through all possible permutations and I think that there are $n!$ of them. So in total the worst total time compliexity is $\Theta(n^2 + n!)$.
```

Comment:
What do you have to do for each of the permutations?

Reply:
Apart from that, we can't indicate time complexity as n^2 + n! -- In polynomial, you have to specify which one is asymptotically bigger. n! = n(n-1)(n-2)...1, which approximately as same as n^n